xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -88,7 +88,11 @@ void CRenderManager::SetVideoSettings(const CVideoSettings& settings)
   }
 }
 
-bool CRenderManager::Configure(const VideoPicture& picture, float fps, unsigned int orientation, int buffers)
+bool CRenderManager::Configure(const VideoPicture& picture,
+                               float fps,
+                               unsigned int orientation,
+                               StreamHdrType hdrType,
```

Comment:
Add `hdrType` parameter is not necessary because already exist in `VideoPicture`